frontend/public/components/overview/networking-overview.tsx

```diff
@@ -51,7 +51,8 @@ const RoutesOverviewListItem: React.SFC<RoutesOverviewListItemProps> = ({ route
   return (
     <li className="list-group-item">
       <ResourceLink kind="Route" name={name} namespace={namespace} />
-      <span className="text-muted">{t('public~Location:')}</span> <RouteLocation obj={route} />
+      <span className="text-muted">{t('public~Location:')}</span>
```

Comment:
I think you want to add a `{' '}` here to keep the space character that existed between the `span` and `RouteLocation`.

Reply:
In the origin source code ,there is a space, I just keep it.  
<span className="text-muted">{'Location: '}</span>
   <RouteLocation obj={route} />